cypress/integration/patch/configure-patch.ts

```diff
@@ -178,6 +178,64 @@ describe("Configure Patch Page", () => {
             .should("eq", $tasks.length);
         });
     });
+    describe("Selecting multiple build variants", () => {
+      it("Selecting multiple build variants should work", () => {
+        cy.get('[data-cy-name="linux-docker"]').click();
```

Comment:
why not use `.dataCy` here?

Reply:
the tag was a `data-cy-name` not a `data-cy` so i cant use `.dataCy` for it.